app/text_buffer.py

```diff
@@ -687,6 +687,7 @@ def applyDocumentUpdate(self, data):
 
   def findCurrentPattern(self, direction):
     localRe = self.findRe
+    offset = self.penCol + direction
```

Comment:
I've moved this function into ci_edit/app/actions.py. Are you up for moving the PR there?